docs/bias_correction.rst

```diff
@@ -62,8 +61,8 @@ The blend factor describes how upstream and downstream flows should be interpola
 
 .. math::
 
-|    BF = \frac{UM}{UM+DM}
-|    TF = (BF*UF) + ((1-BF)*DF)
+    BF = \frac{UM}{UM+DM}
```

Comment:
I'm pretty certain it needs that `|` to make the math work. If you tried rendering it and it was successful without it then alright, but that's something I had to add to get it to work